doc/sphinx/addendum/micromega.rst

```diff
@@ -295,7 +295,7 @@ obtain :math:`-1`. By Theorem :ref:`Psatz <psatz_thm>`, the goal is valid.
    The :tacn:`zify` tactic can be extended with new types and operators by declaring and registering new typeclass instances using the following commands.
    The typeclass declarations can be found in the module ``ZifyClasses`` and the default instances can be found in the module ``ZifyInst``.
 
-.. cmd:: Add Zify @add_zify @one_term
+.. cmd:: Add Zify @add_zify @qualid
```

Comment:
If you really want to restrict what is accepted by `Add Zify`, why didn't you change the parsing rules for the command? If there's a good reason for restricting this not at parsing time but only at interpretation time, then we will need to make a special exception for this in `doc/tools/docgram/common.edit_mlg`.

Reply:
Actually, I did in another recent PR but forgot to update the doc. 
You can see that it is now a `reference` in the mlg code (see https://github.com/coq/coq/blob/master/plugins/micromega/g_zify.mlg)

Reply:
OK, great. Thanks for clarifying!